Session-Cookie Support (#1713)
oauth2-proxy/oauth2-proxy: docs/docs/configuration/overview.md

| `--code-challenge-method` | string | use PKCE code challenges with the specified method. Either 'plain' or 'S256' (recommended) | |
| `--config` | string | path to config file | |
| `--cookie-domain` | string \| list | Optional cookie domains to force cookies to (e.g. `.yourcompany.com`). The longest domain matching the request's host will be used (or the shortest cookie domain if there is no match). | |

> | `--cookie-expire` | duration | expire timeframe for cookie. If set to 0, cookie becomes a session-cookie which does not have \"Expire\" in it. | 168h0m0s |

Reviewer: This is correct right? Might be better to describe the end user behaviour rather than the technical behaviour
```suggestion
| `--cookie-expire` | duration | expire timeframe for cookie. If set to 0, cookie becomes a session-cookie which will expire when the browser is closed. | 168h0m0s |
```

Author: > If set to 0, cookie becomes a session-cookie which will expire when the browser is closed.

Yes, it's true ! Thank you fothe change.
I commited your change.
